VCC-UsageReport/VCC-UsageReport.ps1

```diff
@@ -0,0 +1,143 @@
+<#
+	.SYNOPSIS
+	Veeam Cloud Connect Usage Report
+  
+	.DESCRIPTION
+	This Scropt will Report Cloud Connect Tenant Statistics
+        
+	.EXAMPLE
+	VCC-UsageReport.ps -Server VeeamEM.lan.local -HTTPS:$True -Port 9398 -Authentication Vk9QXHN2Yy12cm6tY2MwMTp2XltKNUNiS2dlIUp6dkQxbkdiZnky
+
+	.EXAMPLE
+	VCC-UsageReport.ps -Server VeeamEM.lan.local -HTTPS:$False -Port 9399 -Authentication Vk9QXHN2Yy12cm6tY2MwMTp2XltKNUNiS2dlIUp6dkQxbkdiZnky
+	
+	.Notes
+	NAME:  VCC-UsageReport.ps
+	LASTEDIT: 08/22/2016
+	VERSION: 1.0
+	KEYWORDS: Veeam, Cloud Connect
+    BASED ON: http://mycloudrevolution.com/2016/08/16/prtg-veeam-cloud-connect-monitoring/
+   
+	.Link
+	http://mycloudrevolution.com/
+ 
+ #Requires PS -Version 3.0  
+ #>
+[cmdletbinding()]
+param(
+    [Parameter(Position=0, Mandatory=$false)]
+    	[String] $Server = "VeeamEM.lan.local",
+	[Parameter(Position=1, Mandatory=$false)]
+		[Boolean] $HTTPS = $True,
+	[Parameter(Position=2, Mandatory=$false)]
+		[String] $Port = "9398",
+	[Parameter(Position=3, Mandatory=$false)]
+		[String] $Authentication = "Vk9QXHN2Yy12cm6tY2MwMTp2XltKNUNiS2dlIUp6dkQxbkdiZnky"
+
+)
+
+#region: Workaround for SelfSigned Cert
+add-type @"
+    using System.Net;
+    using System.Security.Cryptography.X509Certificates;
+    public class TrustAllCertsPolicy : ICertificatePolicy {
+        public bool CheckValidationResult(
+            ServicePoint srvPoint, X509Certificate certificate,
+            WebRequest request, int certificateProblem) {
+            return true;
+        }
+    }
+"@
+[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
+#endregion
+
+#region: Switch Http/s
+if ($HTTPS -eq $True) {$Proto = "https"} else {$Proto = "http"}
+#endregion
```

Comment:
If proto = http, default port is 9399 by default. Suggest adding a check here if port == 9398 && proto == http, and if yes, change port to 9399 as well? Just an idea :)